osu.Game/Screens/SelectV2/PanelBeatmap.cs

```diff
@@ -261,10 +264,6 @@ private void updateDisplay()
             starCounter.Current = (float)starDifficulty.Stars;
 
             difficultyIcon.FadeColour(starDifficulty.Stars > OsuColour.STAR_DIFFICULTY_DEFINED_COLOUR_CUTOFF ? colours.Orange1 : colourProvider.Background5, duration, Easing.OutQuint);
```

Comment:
I think you'll want this to move this to use `starRatingDisplay.DisplayedStars` too.

Reply:
I guess so yeah. I just saw that a lot of colouring logic predicated on star rating was being moved to update and noticed this specific piece of it wasn't, so it looked off to me.

Reply:
Updated in 27ce54960626ab80b9e3b7bfb425d5a362353b21.

There's an unfortunate issue with the new default return from `GetBindableDifficulty`, you'll sere what I mean if you hold ctrl+up at new song select and watch the animation jump back to base difficulty during initial calculation. it's fine after things are cached.